predictive_models/baseline.py

```diff
@@ -0,0 +1,51 @@
+"""A Baseline model
+Source: https://www.tensorflow.org/tutorials/structured_data/time_series
+"""
+
+import tensorflow as tf
+import pandas as pd
+from window_generator import WindowGenerator
+
+
+# pylint: disable=too-few-public-methods
+class Baseline(tf.keras.Model):
+    def __init__(self, label_index=None):
+        super().__init__()
+        self.label_index = label_index
+
+    def call(self, inputs):
+        if self.label_index is None:
+            return inputs
+        result = inputs[:, :, self.label_index]
+        return result[:, :, tf.newaxis]
+
+
+dir_path = "/Users/seppas/Documents/Moody+/SphereDecay/"
+CFD_5 = pd.read_csv(dir_path + "05SphereDecayCFD_V3", header=0)
+CFD_3 = pd.read_csv(dir_path + "03SphereDecayCFD_V3", header=0)
+CFD_1 = pd.read_csv(dir_path + "01SphereDecayCFD_V3", header=0)
+train_df = CFD_5
+val_df = CFD_3
+test_df = CFD_1
```

Comment:
* `CFR_5` -> avoid abbreviations, compromises readability.
* variables `CFR_1`, `CFR_3`, and `CFR_5` are use only here. We can avoid introducing them and assign the values directly to `train_df`, `val_df`, and `test_df `.
* we should use `os.path.join` for generating paths, it's safer: `dir_path + "05SphereDecayCFD_V3"` -> `os.path.join(dir_path, "05SphereDecayCFD_V3")`
* avoid hard coding file names (`"05SphereDecayCFD_V3"`, ...), they should be provided by the use or be read from a config file or something similar.